website/docs/core-concepts/validate/validate.mdx

```diff
@@ -55,41 +59,41 @@ The command checks and validates the following:
 - Schema: if all sections in all YAML manifest files are correctly configured and have valid data types
 
 - Misconfiguration and duplication of components in stacks. If the same Atmos component in the same Atmos stack is
-  defined in more than one stack manifest file, and the component configurations are different, an error message will
-  be displayed similar to the following:
+defined in more than one stack manifest file, and the component configurations are different, an error message will
+be displayed similar to the following:
 
-  <Terminal title="atmos validate stacks">
-    ```console
-    The Atmos component 'vpc' in the stack 'plat-ue2-dev' is defined in more than one
-    top-level stack manifest file: orgs/acme/plat/dev/us-east-2-extras, orgs/acme/plat/dev/us-east-2.
+<Terminal title="atmos validate stacks">
+```console
```

Comment:
Changing the indention now means this is no longer associated with the preceeding bullet point.